### PR TITLE
Tobias Hagberg, Jan2021 -Team Tigers

### PR DESCRIPTION
**Pull request by TobiasHagberg**
Tobias Hagberg, Jan2021 -Team Tigers

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+const factDefintion = document.getElementById("fact-definition");
+
+
+didYouKnow.addEventListener("click", ()=> {
+  factDidYouKnow.classList.toggle("selected");
+})
+
+defintionClick.addEventListener("click", ()=> {
+  factDefintion.classList.toggle("selected");
+})
```
Comment (HIPPIEKICK) [inline code/script.js:18]: Use semicolons either everywhere or not at all

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,141 @@
...
+.accordion {
+  font-family: 'Fraunces', serif;
+}
+
+.did-you-know {
+  background-color: #eee;
+  color: #444;
+  cursor: pointer;
+  padding: 18px;
+  width: 100%;
```
Comment (HIPPIEKICK) [inline code/style.css:97]: These two together makes the content grow too wide:
Skärmavbild 2021-02-09 kl  17 13 03

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Hi Tobias! Please have a look at my comments in your HTML file regarding the different tags, as well as the responsiveness of your page. Take a look at the [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you want to recap conventions. Check out the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475) for project-specific tips.

**Comment by KarolCislowski** [comment]
Hello, it's your project. Do you remember me? I may be imperfect, but I believe you can still change that before you sit down to your final masterpiece!
image

**Comment by TobiasHagberg** [comment]
Hi Matilda and Karol,
Changes has been made to this weeks project and lesson learned! :)

Kind regards
Tobias

**Review by KarolCislowski — APPROVED** [review]
Nice page 🎉

**Pull request closed without merging**